tensornetwork/__init__.py

```diff
@@ -1,6 +1,13 @@
 from __future__ import absolute_import
+
 from tensornetwork.network import TensorNetwork
-from tensornetwork.network_components import Node, Edge, CopyNode
+from tensornetwork.network_components import Node, Edge, CopyNode, BaseNode
+#pylint: disable=line-too-long
+from tensornetwork.network_operations import conj, transpose, split_node, split_node_qr, split_node_rq, split_node_full_svd, reachable, check_connected, check_correct, get_all_nodes, get_all_edges
+
+from tensornetwork.network_components import contract, contract_copy_node, contract_between, outer_product, outer_product_final_nodes, contract_parallel, flatten_edges, get_all_nondangling, flatten_all_edges, flatten_edges_between, get_parallel_edges, get_shared_edges, _contract_trace
```

Comment:
Don't include the private `_contract_trace` method.

Reply:
removed